scripts/set_plugin_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PLUGIN_FILE_PATH="../crates/snforge-scarb-plugin/Scarb.toml"
+
+VERSION=$(grep version "$PLUGIN_FILE_PATH" | cut -d '"' -f 2)
+
+STD_FILE_PATH="../snforge_std/Scarb.toml"
```

Comment:
```suggestion
PLUGIN_FILE_PATH="../crates/snforge-scarb-plugin/Scarb.toml"
STD_FILE_PATH="../snforge_std/Scarb.toml"

VERSION=$(grep version "$PLUGIN_FILE_PATH" | cut -d '"' -f 2)
```
nitpick

btw https://en.wikipedia.org/wiki/STD_(disambiguation) ( ͡° ͜ʖ ͡°)

Let's maybe use full `SNFORGE_STD_PATH` name